Tribler/Core/Modules/restapi/events_endpoint.py

```diff
@@ -18,6 +19,14 @@ class EventsEndpoint(resource.Resource):
     - events_start: An indication that the event socket is opened and that the server is ready to push events.
     - search_result_channel: This event dictionary contains a search result with a channel that has been found.
     - search_result_torrent: This event dictionary contains a search result with a torrent that has been found.
+    - upgrader_started: An indication that the Tribler upgrader has started.
+    - upgrader_finished: An indication that the Tribler upgrader has finished.
+    - upgrader_tick: An indication that the state of the upgrader has changed. The dictionary contains a human-readable
+      string with the new state.
+    - watch_folder_corrupt_torrent: This event is emitted when a corrupt .torrent file in the watch folder is found.
```

Comment:
That name is really weird. It should definitely be something like `NTFY_WATCH_FOLDER_CORRUPT_TORRENT`. How did this get through in the first place?